pkg/config/config_template.yaml

```diff
@@ -273,8 +273,14 @@ api_key:
 # dogstatsd_port: 8125
 #
 # The host to bind to receive external metrics (used only by the dogstatsd
```

Comment:
this should be updated to reflect that it's used by both the Dogstatsd server and the APM (trace?) Agent :)

Reply:
👍 Done.

Reply:
did you actually update this? 🤔 Still mentioning `(used only by the dogstatsd server for now)`

Reply:
I'm so sorry. I got confused here. Fixed.